### ISSUE TITLE
docstring spread.interfaces incorrect

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6299]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#6299|
|Type|defect|
|Created|2013-02-12 03:03:02Z|

`t.spread.interfaces` is missing a copyright header and states its unused but its imported and used in `spread.jelly`?

Attachments:

* [added_copyright_and_corrected_docstring.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/97c/97c81eb9268ae91c640b009772696e55f51231b8/5496cb98114c4ba9a78a0b755b8d33159436643b.patch) (488 bytes) - added by dhanush on 2013-03-28 17:40:58Z - Added copyright and corrected docstring. But test_case_name is still missing. 

<details><summary>Searchable metadata</summary>

```
trac-id__6299 6299
type__defect defect
reporter__thijs thijs
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation_easy documentation easy
time__1360638182000000 1360638182000000
changetime__1364924448000000 1364924448000000
version__None None
owner__thijs thijs
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_2"></a>|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

Thanks for your patch dhanush, I'll apply parts of it. Make sure you check the instructions on [TwistedDevelopment#SubmittingaPatch how to supply] a patch in the future.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [37913]) Apply patch: Add missing copyright header and remove statement that the module is not being used.

Author: dhanush
Reviewer: thijs
Fixes: [#6299](https://github.com/twisted/twisted/issues/6299)

**Comment by twisted-trac** [comment]
|<img alt="dhanush's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_4"></a>|dhanush commented|
|-|-|

Hi thijs. I will surely stick to the norms on submitting a patch the next time. Thanks for your advice! :)
